### PR TITLE
Configuration Examples for tail and json parsing

### PR DESCRIPTION
**Pull request by derjohn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,10 +86,15 @@ Reading from the JSON formatted log files with `in_tail` and wildcard filenames:
...
   read_from_head true
+  tag kubernetes.*
+  <parse>
+    @type json
+    time_key time
+    time_type string
+    time_format "%Y-%m-%dT%H:%M:%S.%NZ"
+    keep_time_key false
+  </parse>
 </source>
```
Comment (richm) [inline README.md]: We should support CRI-o log file format too with our example.  Please change the `<parse>` block to something like this (or I can change the commit directly if you don't mind):
```
  <parse>
    @type multi_format
    <pattern>
      format json
      time_key time
      time_type string
      time_format "%Y-%m-%dT%H:%M:%S.%NZ"
      keep_time_key false
    </pattern>
    <pattern>
      format regexp
      expression /^(?<time>.+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/
      time_format '%Y-%m-%dT%H:%M:%S.%N%:z'
      keep_time_key false
    </pattern>
  </parse>
```

**Comment by derjohn** [comment]
Hey, cool, I wasn't aware about that. I'll sned a patch.

BTW: IIRC there are some open issues about json logging you can close now.

**Comment by richm** [comment]
> BTW: IIRC there are some open issues about json logging you can close now.

which ones are those?

**Comment by derjohn** [comment]
Oh, it wasn't here, I've been confused. I just found them when analyzing my issue. 

https://github.com/fluent/fluentd-kubernetes-daemonset/issues/273
https://github.com/fluent/fluentd-kubernetes-daemonset/issues/324
https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/145
....

All closed, so go ahead! ;)

Thx,
derjohn

File: README.md (hunk 1 above)
Comment (fabio-s-franco) [reply README.md:113]: I have been search all over the place for this. Not nearly reachable enough.

**Pull request merged**